.gitignore

```diff
@@ -55,3 +56,7 @@ desktop.ini
 # Jetbrains stuff
 .idea/**
 cmake-build-*/
+
+# Install folders
+out/
+dist/
```

Comment:
These two directories are created only in the root of the source tree, right? So they can be prepended with a `/` prefix to limit their scope there:
```
/out/
/dist/
```